migrate/migrations/2022-12-11T09:17:35_init.ts

```diff
@@ -38,8 +38,15 @@ export async function up(db: Kysely<Database>): Promise<void> {
     .addColumn("modified_at", "varchar(255)")
     .addPrimaryKeyConstraint("users_tenants", ["id", "tenant_id"])
     // Added in later migration
-    .addColumn("linked_to", "varchar(255)", (col) =>
-      col.references("users.id").onDelete("cascade"),
+    // .addColumn("linked_to", "varchar(255)", (col) =>
+    //   col.references("users.id").onDelete("cascade"),
+    // )
+    .addColumn("linked_to", "varchar(255)")
+    .addForeignKeyConstraint(
+      "linked_to_constraint",
+      ["linked_to", "tenant_id"],
+      "users",
+      ["id", "tenant_id"],
```

Comment:
I did this here yesterday https://github.com/sesamyab/auth/pull/357/files 

I think you want a `(cb) => cb.onDelete("cascade"),`

```suggestion
    .addColumn("linked_to", "varchar(255)")
    .addForeignKeyConstraint(
      "linked_to_constraint",
      ["linked_to", "tenant_id"],
      "users",
      ["id", "tenant_id"],
      (cb) => cb.onDelete("cascade"),
```

I can merge this in and then my PRs will have the remaining changes left over :+1: